Enable the Lua Loader which fixes #78

kmoore134 opened the pull request.

kmoore134 committed:
Enable the Lua Loader which fixes #78

kmoore134 committed:
Add TrueOS brand and cleanup defaults file

pkgdemon reviewed (commented):
@kmoore134 How will this affect the previously committed logo files for Trident, and GhostBSD?  Won’t you need to update those as well?

kmoore134 commented:
Yep, which is why I've not asked for review yet. Still working those bits
out, but in meantime this is helping me build isos for testing.

--
Kris Moore
Vice President of Engineering
iXsystems

On Sat, Jun 9, 2018, 11:14 PM Joe Maloney <notifications@github.com> wrote:

> *@pkgdemon* commented on this pull request.
>
> @kmoore134 <https://github.com/kmoore134> How will this affect the
> previously committed logo files for Trident, and GhostBSD? Won’t you need
> to update those as well?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/trueos/trueos/pull/79#pullrequestreview-127383943>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1_IJgnG5I-S65LaI8J_5uFF-52AlEGks5t7I8JgaJpZM4UgfWu>
> .
>

kevans91 committed:
lualoader: Allow brand-*.lua for adding new brands

dteske@, I believe, had originally pointed out that lualoader failed to
allow logo-*.lua for new logos to be added. When correcting this mistake, I
failed to do the same for brands.

Correct the sub-mistake: creating new brands is almost identical to creating
new logos, except one must use `drawer.addBrand` and 'graphic' is the only
valid key for a branddef at the moment.

While here, I've added `drawer.default_brand` to be set to name of brand to
be used (e.g. 'fbsd', project default).

Eventually this whole goolash will be documented.

Reported by:	kmoore, iXsystems

kmoore134 committed:
Add TrueOS specific brand .lua file

kmoore134 commented:
Joe, please take another look. I've pulled in patch which adds support for the brand-* files now. I'm not planning on converting all the brands over, will let each project do their own to make sure it lives up to their standards. However the default "TrueOS" one works now.
